children/schema.py

```diff
@@ -22,11 +34,10 @@ class ChildInput(graphene.InputObjectType):
     birthdate = graphene.Date(required=True)
 
 
-# this class cannot be named just RelationshipType because of graphene-django
-# https://github.com/graphql-python/graphene-django/issues/185
-class RelationshipObjectType(DjangoObjectType):
+class RelationshipNode(DjangoObjectType):
```

Comment:
yes, very nice! we should indeed call them `...Node` if they implement `Node interface` 👍 should be part of the GraphQL guidelines.